include/ac_koinobori.h

```diff
@@ -3,13 +3,30 @@
 
 #include "types.h"
 #include "m_actor.h"
+#include "c_keyframe.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
 extern ACTOR_PROFILE Koinobori_Profile;
 
+typedef void (*KOINOBORI_PROC)(ACTOR*, GAME*);
+
+/* sizeof(KOINOBORI_ACTOR) == 0x2DC */
+typedef struct actor_koinobori_s{
+    /* 0x00 */ ACTOR actor_class;
```

Comment:
Super minor nitpick, this is missing a zero relative to other member offsets. You don't have to fix this if you don't feel like it.

Reply:
Extra zero added for consistency